### PR TITLE
JBIDE-27453: Update 4.17.0.AM1 Target Platform for 2020-09 RC1

### PR DESCRIPTION
**Pull request by sbouchet**
Signed-off-by: Stephane Bouchet <sbouchet@redhat.com>

### CODE REVIEW

File: jbtcentraltarget/multiple/jbtcentral-multiple.target [hunk 1]
```diff
@@ -70,6 +70,8 @@
       <unit id="org.aspectj.weaver" version="1.9.6.202004271746"/>
       <unit id="org.eclipse.equinox.weaving.sdk.feature.group" version="1.2.0.202004271746"/>
       <unit id="org.eclipse.equinox.weaving.hook" version="1.2.200.v20180827-1235"/>
+
```
Comment (sbouchet) [inline jbtcentraltarget/multiple/jbtcentral-multiple.target:73]: updated TP to add this missing bundle causing installs to fails.

Comment (jeffmaury) [reply jbtcentraltarget/multiple/jbtcentral-multiple.target:73]: yedit is to be removed so I think we should not add it here but remove it from the studio definition

**Review by jeffmaury — CHANGES_REQUESTED** [review]

File: jbtcentraltarget/multiple/jbtcentral-multiple.target (hunk 1 above)
Comment (sbouchet) [reply jbtcentraltarget/multiple/jbtcentral-multiple.target:73]: without yedit on central, spring IDE cannot be installed, causing our jobs to fails:
https://dev-platform-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Devstudio/view/devstudio_master/job/jbosstools-composite-install_master/lastFailedBuild/parsed_console/job/jbosstools-composite-install_master/101106/parsed_console/log_content.html#ERROR2

**Comment by jeffmaury** [comment]
Then it's an issue because it will interfere with the YAML support that I just added to Quarkus Tools as it will launch the YEdit editor and not the one from WWD. Did we move to SpringIDE 4 ?

**Comment by sbouchet** [comment]
@jeffmaury : no we still are using SpringIDE 3. We already created a JIRA for moving to SpringIDE4 : https://issues.redhat.com/browse/JBIDE-27324

**Comment by sbouchet** [comment]
@jeffmaury: i also suspect that spring 4 still use Yedit editor as yaml editor..

**Comment by jeffmaury** [comment]
Yes just checked the STS4 repo. So we have an issue that we need to investigate

**Review by jeffmaury — APPROVED** [review]

**Pull request merged**